From 688001fd5b57e51e0a4b3e23ca25fe7ccfa1fd4a Mon Sep 17 00:00:00 2001
From: Lubosz Sarnecki <lubosz@gmail.com>
Date: Sun, 3 Aug 2014 15:11:10 +0200
Subject: [PATCH] msys2: fix gir-scanner

* add CC flags to gir-scanner

* add clutter and base plugins includes to gir
---
 clutter-gst/Makefile.am | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/clutter-gst/Makefile.am b/clutter-gst/Makefile.am
index f9d4ecd..cb02c52 100644
--- a/clutter-gst/Makefile.am
+++ b/clutter-gst/Makefile.am
@@ -139,6 +139,7 @@ if HAVE_INTROSPECTION
 ClutterGst-@CLUTTER_GST_API_VERSION@.gir: $(INTROSPECTION_SCANNER) libclutter-gst-@CLUTTER_GST_API_VERSION@.la
 	$(QUIET_GEN) \
 		GST_PLUGIN_SYSTEM_PATH_1_0="" GST_PLUGIN_PATH_1_0="" GST_REGISTRY_UPDATE=no DISPLAY="" \
+		CC="$(CC)" \
 		$(INTROSPECTION_SCANNER) -v \
 		--namespace ClutterGst --nsversion=@CLUTTER_GST_API_VERSION@ \
 		$(INCLUDES) \
-- 
2.0.4

